### ISSUE TITLE
Game maker isometric maps exporting as orthogontal.

### ISSUE THREAD

**Issue by strelokhalfer** [bjorn/tiled#1435]
Tiled is exporting isometric maps as orthogontal.
Standart isometric
![default](https://cloud.githubusercontent.com/assets/4334058/22196413/23ff96e6-e15e-11e6-9d41-ecdad317e1a1.png)
Staggered by Y
![default](https://cloud.githubusercontent.com/assets/4334058/22196450/50bb7466-e15e-11e6-8814-33c74680952f.png)
Staggered by X
![default](https://cloud.githubusercontent.com/assets/4334058/22196464/5bf3214e-e15e-11e6-95fb-3bbead13c5d0.png)
In GameMaker(each export do same, box drawed in gm room editor)
![default](https://cloud.githubusercontent.com/assets/4334058/22196504/7ebd4af6-e15e-11e6-9491-b6d465c26530.png)
If we set orientation to orthogantal we see...
![default](https://cloud.githubusercontent.com/assets/4334058/22196637/362d8e08-e15f-11e6-90fd-ec5b13c24347.png)

**Comment by bjorn** [comment]
Right, support for isometric maps (or any other map than orthogonal, really) is currently missing for the GameMaker export. I'm sure it could be added though.

**Comment by The-Evil-Dice** [comment]
@strelokhalfer If you're still looking to do this, I've made a tool to do this for myself.
It's nothing fancy just a command line jar, but given how many people I've seen wanting to get around this I figured I may aswell upload it.
And @bjorn , you're more than welcome to use my logic as a quick fix, I'm sure you could do better in the long run though.

https://github.com/The-Evil-Dice/TiledCSVtoGMS/blob/master/README.md

**Comment by bjorn** [comment]
@The-Evil-Dice Are you not at all inclined to look into compiling Tiled yourself and possibly making this an enhancement to the existing GameMaker export functionality? There, you could probably use the available MapRenderer subclasses to make this work for staggered maps with any setup as well as hexagonal maps.

**Comment by The-Evil-Dice** [comment]
@bjorn I'll gladly have a stab, though I'm not very fluent in C++.

**Comment by bjorn** [comment]
Just putting a backlink here, to note that this was also mentioned on the forum:

https://discourse.mapeditor.org/t/problem-with-isometric-maps-in-game-maker-studio/2702
